src/PiGui.cpp

```diff
@@ -284,7 +284,7 @@ void *PiGui::makeTexture(const std::string &filename, unsigned char *pixels, int
 	Pi::renderer->CheckRenderErrors(__FUNCTION__, __LINE__);
 	// nasty bit as I invoke the TextureGL
 	Graphics::OGL::TextureGL *pGLTex = reinterpret_cast<Graphics::OGL::TextureGL*>(pTex);
-	Uint32 result = pGLTex->GetTexture();
+	Uint32 result = pGLTex->GetTextureID();
```

Comment:
Question : should we keep the Uint32 type (which I assume is "homemade" or use the uint32_t type ? I see the GetTextureId() declaration uses the latter.